### PR TITLE
[sfntedit] Fix sfntedit bug 160

### PR DESCRIPTION
**Pull request by readroberts**
Made changes to continue if a table is missing (a very small change), and to not change the output file if no changes are made.

Added test case to assert that a warning is shown for the attempt to delete or extract a missing table, but that the tool does continue and operates on  another table.

### CODE REVIEW

**Comment by readroberts** [comment]
@msousa @cjchapman  @frankrolf I added a new option partly because that was part of the request in issue #160, and no other comment was made, and partly because it seems like a good idea to me. When I use sfntedit, I want it to work entirely, or not all and to exit with an error. In my work, by far the most common case of a missing table is when I mis-type a tag, and I want a clear indication that I did something wrong. I see fixing this issue as a concession to a particular user's workflow for a rare case, in the spirit of allowing developers to work as they want. Do you indeed think  that a skipping missing tables without an error should be the default behavior?

**Comment by miguelsousa** [comment]
Yes, no option. Instead of an error make it a warning.

**Comment by readroberts** [comment]
Will do. Removing the option is an easy change. Can you confirm that in the case fo a missing table, sfntedit should with exit with no error code? I think this is appropriate for just a warning.

**Comment by miguelsousa** [comment]
Yes, exit with `0` and issue a warning.

**Comment by readroberts** [comment]
Note that the changed logic was force-pushed, so you will need to update your copy of this branch.

File: c/sfntedit/source/main.c [hunk 1]
```diff
@@ -976,6 +985,9 @@ static void sfntCopy(void) {
         totalsum += checksum;
     }
 
+        if (!changed)
```
Comment (cjchapman) [inline c/sfntedit/source/main.c]: please fix indentation here

Comment (cjchapman) [reply c/sfntedit/source/main.c]: (otherwise the changes look OK to me)

**Comment by readroberts** [comment]
Fixed white-space issue.

**Pull request merged**